### PR TITLE
Add riscv64 target

### PR DESCRIPTION
**Pull request by luyahan**
Just only add riscv64 config for no-asm.

### CODE REVIEW

**Review by levitte — APPROVED** [review]

**Comment by levitte** [comment]
Formality: I agree this is trivial

File: Configurations/10-main.conf [hunk 1]
```diff
@@ -804,7 +804,13 @@ my %targets = (
         perlasm_scheme   => "64",
         multilib         => "64",
     },
-
+
```
Comment (t8m) [inline Configurations/10-main.conf:807]: Please no spaces at the end of line.

File: Configurations/10-main.conf [hunk 2]
```diff
@@ -804,7 +804,13 @@ my %targets = (
...
         multilib         => "64",
     },
-
+    
+    # riscv64 below refer to contemporary RISCV Architecture
+    # specifications,
+    "linux64-riscv64" => {
+        inherit_from     => [ "linux-generic64"],
+        perlasm_scheme   => "linux64",
+    },
```
Comment (t8m) [inline Configurations/10-main.conf:813]: There should be empty line after this one to separate it from different architecture.

File: Configurations/10-main.conf [hunk 3]
```diff
@@ -804,7 +804,13 @@ my %targets = (
         perlasm_scheme   => "64",
         multilib         => "64",
     },
-
+    
+    # riscv64 below refer to contemporary RISCV Architecture
```
Comment (t8m) [inline Configurations/10-main.conf:808]: `refers`?

**Review by t8m — CHANGES_REQUESTED** [review]
Also I am not quite sure about the CLA: trivial.

Citing our CLA policy:
`A submission is trivial if it is considered trivial under copyright law. Since we are not lawyers, we place the bar for trivial contributions very high. For example: corrections of grammatical or typographical errors (including misspelled function names in manual pages), simple whitespace changes and in some cases one-line bugfixes might be accepted as trivial without requiring a CLA.`
This seems to be a bit more than just one-line bug fix.

**Comment by levitte** [comment]
> Also I am not quite sure about the CLA: trivial.

Okie.  @luyahan, would you mind submitting a CLA, then?

**Comment by luyahan** [comment]
> > Also I am not quite sure about the CLA: trivial.
> 
> Okie.  @luyahan, would you mind submitting a CLA, then?

OK,i will to try sign it. Thanks

**Comment by luyahan** [comment]
> > Also I am not quite sure about the CLA: trivial.
> 
> Okie. @luyahan, would you mind submitting a CLA, then?

I have sended email with signed pdf file to  legal@opensslfoundation.org

File: Configurations/10-main.conf (hunk 2 above)
Comment (luyahan) [reply Configurations/10-main.conf:812]: done

**Review by levitte — APPROVED** [review]

File: Configurations/10-main.conf [hunk 4]
```diff
@@ -804,6 +804,13 @@ my %targets = (
         perlasm_scheme   => "64",
         multilib         => "64",
     },
+
```
[the remark this answers is not in the document]
Comment (luyahan) [reply Configurations/10-main.conf]: Thanks your review. I had fixed it.

**Review by paulidale — APPROVED** [review]

**Comment by t8m** [comment]
Merged to master branch. Thank you for the contribution!

**Pull request closed without merging**